aspnetcore/blazor/get-started.md

```diff
@@ -15,91 +15,115 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
 
 [!INCLUDE[](~/includes/blazorwasm-preview-notice.md)]
 
-Get started with Blazor:
+To get started with Blazor, follow the guidance for your choice of tooling:
 
-1. Install the [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1).
+# [Visual Studio](#tab/visual-studio)
 
-1. Optionally install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template:
-   * Install the [.NET Core 3.1.102 or later (Preview) SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1).
-   * Run the following command in a command shell. The [Microsoft.AspNetCore.Components.WebAssembly.Templates](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.Templates/) package has a preview version while Blazor WebAssembly is in preview.
+1. Install [Visual Studio 2019 version 16.4 or later](https://visualstudio.microsoft.com/vs/preview/) with the **ASP.NET and web development** workload.
+
+   When installing Visual Studio 2019 16.6 Preview 2 or later, the .NET Core 3.1.300 SDK is automatically installed with Visual Studio. The 3.1.300 SDK automatically provides the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template to create Blazor WebAssembly apps.
+   
+   If installing a version of Visual Studio earlier than version 16.6 Preview 2, optionally install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template. Run the following command in a command shell. The [Microsoft.AspNetCore.Components.WebAssembly.Templates](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.Templates/) package has a preview version while Blazor WebAssembly is in preview.
```

Comment:
For folks that want to get started with Blazor WebAssembly and VS I think we should recommend people use the latest preview of 16.6 and cut any instructions for working with Blazor WebAssembly and older VS versions.